### DISCUSSION TITLE
vmagent

### DISCUSSION THREAD

**Discussion by gusah009** [BDD-CLUB/challenge-every-week-posting#486] [1. 개발 관련 글 모음]
요즘(?) 핫한 VictoriaMetrics의 vmagent에 대해 공부해봤습니다.

얼른 go언어도 공부해서 contribute도 해보겠습니다.

https://gusah009.github.io/infra/2024-05-01-vmagent/

**Comment by 02ggang9** [comment]
이번 클라우드 컴퓨팅 텀프에 꼭 붙여보도록 하겠습니다! 

좋은 글 감사합니다~
